src/status_im/utils/db.cljs

```diff
@@ -0,0 +1,27 @@
+(ns status-im.utils.db
+  (:require [clojure.string :as string]
+            [cljs.spec.alpha :as spec]
+            [status-im.js-dependencies :as dependencies]))
+
+(defn address? [s]
+  (.isAddress dependencies/Web3.prototype s))
+
+(defn hex-string? [s]
```

Comment:
this should probably be done in a utils refactoring but I don't think it should be part of this PR it is already big enough